pom.xml

```diff
@@ -146,7 +146,12 @@
 			<dependency>
 				<groupId>org.slf4j</groupId>
 				<artifactId>slf4j-api</artifactId>
-				<version>1.7.8</version>
+				<version>1.7.10</version>
+			</dependency>
+			<dependency>
+				<groupId>org.slf4j</groupId>
+				<artifactId>slf4j-simple</artifactId>
+				<version>1.7.10</version>
```

Comment:
The line you commented is only dependency management to declare the version, all actual uses I added are in the test scope